spec/rubocop/runner_spec.rb

```diff
@@ -60,6 +60,40 @@ def INVALID_CODE
         END
       end
     end
+
+    context 'if -s/--stdin is used with an offense' do
+      let(:options) do
+        {
+          formatters: [['progress', formatter_output_path]],
+          stdin: <<-END.strip_indent
```

Comment:
The options will blow up if `--stdin` is used without exactly one path, but in the runner it really doesn't matter. With the `stdin` option it's going to use that same source for every path passed in, which in this case just matches the single `example.rb` created above.